src/client/features/navigation.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import type { Feature } from '../types';
+
+/**
+ * Client-side feature for browser navigation.
+ */
+export const navigate: Feature = {
+	id: 'core/navigate',
+	name: __( 'Navigate Browser' ),
+	description: __( 'Navigates the browser to a specified URL.' ),
+	type: 'tool',
+	location: 'client',
+	categories: [ 'core', 'navigation' ],
+	input_schema: {
+		type: 'object',
+		properties: {
+			url: {
+				type: 'string',
+				description: __(
+					'The URL to navigate to (can be absolute or relative).'
+				),
+			},
+		},
+		required: [ 'url' ],
+	},
+	callback: ( args: { url: string } ) => {
```

Comment:
Minor: The logic on this function has an issue:
My dev site is at http://localhost:6888/site-wp-dev/.
I said "Navigate to the posts list" the tool invoked was:
{
  "url": "/wp-admin/edit.php"
}
Which seems ok, but the redirect was to http://localhost:6888/wp-admin/edit.php instead of http://localhost:6888/site-wp-dev/wp-admin/edit.php.

Reply:
Fixed in https://github.com/Automattic/wp-feature-api/pull/18/commits/9bb6cca4f05548b781dbe06dd665722d0c23bef0